docs/website/content/v0.3/en/guides/cloud/aws.md

```diff
@@ -2,18 +2,225 @@
+```
+
+### Create a Security Group
+
+```bash
+aws ec2 create-security-group \
+    --region ${REGION} \
+    --group-name talos-aws-tutorial-sg \
+    --description "Security Group for EC2 instances to allow ports required by Talos"
+```
+
+Allow all internal traffic within the same security group:
+
+```bash
+aws ec2 authorize-security-group-ingress \
+    --region ${REGION} \
+    --group-name talos-aws-tutorial-sg \
+    --protocol all \
+    --port 0 \
+    --group-id ${SECURITY_GROUP} \
+    --source-group ${SECURITY_GROUP}
+```
+
+Expose the Talos API and Kubernetes API:
+
+```bash
+aws ec2 authorize-security-group-ingress \
+    --region ${REGION} \
+    --group-name talos-aws-tutorial-sg \
+    --protocol tcp \
+    --port 6443 \
+    --cidr 0.0.0.0/0 \
+    --group-id ${SECURITY_GROUP}
+aws ec2 authorize-security-group-ingress \
+    --region ${REGION} \
+    --group-name talos-aws-tutorial-sg \
+    --protocol tcp \
+    --port 50000-50001 \
+    --cidr 0.0.0.0/0 \
+    --group-id ${SECURITY_GROUP}
+```
 
-> An official Terraform module is currently being developed, stay tuned!
+### Talos Machine Configuration Files
+
+#### Generating Base Configurations
+
+Using the DNS name of our loadbalancer, generate the base configuration files for the Talos machines:
+
+```bash
+$ osctl config generate talos-k8s-aws-tutorial https://${LB_DNS_NAME}
+created init.yaml
+created controlplane.yaml
+created join.yaml
+created talosconfig
+```
+
+At this point, you can modify the generated configs to your liking.
+
+#### Validate the Configuration Files
+
+```bash
+$ osctl validate --config init.yaml --mode cloud
+init.yaml is valid for cloud mode
+$ osctl validate --config controlplane.yaml --mode cloud
+controlplane.yaml is valid for cloud mode
+$ osctl validate --config join.yaml --mode cloud
+join.yaml is valid for cloud mode
+```
+
+### Create the EC2 Instances
+
+#### Create the Bootstrap Node
+
+```bash
+aws ec2 run-instances \
+    --region ${REGION} \
+    --image-id ${AMI} \
+    --count 1 \
+    --instance-type t3.small \
+    --user-data file://init.yaml \
+    --subnet-id ${SUBNET} \
+    --security-group-ids ${SECURITY_GROUP}
+```
+
+#### Create the Remaining Control Plane Nodes
+
+```bash
+aws ec2 run-instances \
+    --region ${REGION} \
+    --image-id ${AMI} \
+    --count 2 \
+    --instance-type t3.small \
+    --user-data file://controlplane.yaml \
+    --subnet-id ${SUBNET} \
+    --security-group-ids ${SECURITY_GROUP}
+```
+
+#### Create the Worker Nodes
+
+```bash
+aws ec2 run-instances \
+    --region ${REGION} \
+    --image-id ${AMI} \
+    --count 3 \
+    --instance-type t3.small \
+    --user-data file://join.yaml \
+    --subnet-id ${SUBNET} \
+    --security-group-ids ${SECURITY_GROUP}
+```
+
+### Configure the Load Balancer
+
+```bash
+aws elbv2 create-target-group \
+    --region ${REGION} \
+    --name talos-aws-tutorial-tg \
+    --protocol TCP \
+    --port 6443 \
+    --vpc-id ${VPC}
```

Comment:
do we want to add a target group for `tcp/50000` yet in prep for the upcoming changes?

Reply:
I don't think so. I thought we decided on client-side load balancing?